shell/browser/ui/message_box_win.cc

```diff
@@ -107,11 +107,11 @@ DialogResult ShowTaskDialogUTF16(NativeWindow* parent,
 
   // TaskDialogIndirect doesn't allow empty name, if we set empty title it
   // will show "electron.exe" in title.
-  std::u16string app_name = base::UTF8ToUTF16(Browser::Get()->GetName());
+  std::wstring app_name = base::UTF8ToWide(Browser::Get()->GetName());
   if (title.empty())
     config.pszWindowTitle = app_name.c_str();
   else
-    config.pszWindowTitle = title.c_str();
+    config.pszWindowTitle = base::UTF16ToWide(title).c_str();
```

Comment:
same here, should the args to this function be `std::wstring`?

Reply:
Done in 1594c549332d4245911ee0d33da3e5196f18d821, but I stopped at the file level scope and didn't deut16ize any further. As with the other comment, I'd like to understand better when each is preferred.